internal/monitors/docker/docker.go

```diff
@@ -67,6 +68,17 @@ type Config struct {
 	ExcludedImages []string `yaml:"excludedImages"`
 }
 
+// Validate docker-container-stats monitor config
+func (c *Config) Validate() error {
+	for _, group := range c.ExtraGroups {
```

Comment:
This is validated already by the extra metrics filter (actually we don't error if the group name is invalid, we just warn per @keitwb) so I don't think we need to do it here. But it does mean that we can't assume that the groups in `c.ExtraGroups` are valid anywhere else.